docs/integrations/rest-api.mdx

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_position: 3
+title: REST API
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Using the REST API to run flows
+
+In this guide we'll demonstrate how you can use the REST API to integrate your flow into existing systems. All API 
+calls will be exemplified using the command line tool cURL, but any tool you're using that has support for making 
+arbitrary REST API calls will work similarly.
+
+<br/>
+
+## Prerequisites
+
+You need to have a [Cradl AI](https://app.cradl.ai/) account and have created a flow. If you haven't, go to 
+[flows](https://app.cradl.ai/flows) and create one.
+
+<br/>
+
+## Step 1: Acquire API credentials
+
+Go to the [API tab](https://app.cradl.ai/appclients) in Cradl and use existing credentials or create new ones. Read 
+more on how to use the credentials to make authenticated requests [here](../reference/authentication.md)
+
+<br/>
+
+## Step 2: Run a flow with a document
+
+Enable the REST API input integration on your flow by going to the flow builder in Cradl AI and locate the `workflowId` 
+that will be required to run a flow.
```

Comment:
From a technical point of view it doesn't need to be enabled as it's always enabled. It is up for discussion whether or not it should follow the same convention as the other integrations in Cradl. @staalezh @magnurud @stinebuu 
 thoughts?

Reply:
I think it makes sense to follow the same convention as the other integrations. How about
```suggestion
Enable the REST API input integration on your flow by choosing "REST API" in the Input drop-down menu in the flow builder in Cradl AI. 
Then, copy the `flowId`. This will be required to run a flow.
```

Reply:
Will `workflowId` be renamed to `flowId` from API calls as well?

Reply:
> Will `workflowId` be renamed to `flowId` from API calls as well?

Eventually yes, but we are stuck with `workflowId` for a couple of months

Reply:
Alright. Let's just make sure we don't break any integrations.That change is going to affect dynamic dropdowns in Zapier, for instance.